modules/badge/css/README.md

```diff
@@ -0,0 +1,62 @@
+# Canvas Kit CSS Badge
+
+The count badge provides a quantity-based summary with dynamic values.
+
+## Installation
+
+```sh
+yarn add @workday/canvas-kit-css
+```
+
+or
+
+```sh
+yarn add @workday/canvas-kit-css-badge
+```
+
+Add your `node_modules` directory to your SASS `includePaths`. You will then be able to import
+`index.scss`.
+
+```scss
+@import '~@workday/canvas-kit-css-badge/index.scss';
+```
+
+## Accessibility
+
+If the count is live-updated (as in the case of notifications) and not static, please add
+`aria-live="polite"` to inform screen readers to announce updated information. When this attribute
+is set the screen reader will announce changes when the user is idle.
+
+```html
+<span class="wdc-count-badge" aria-live="polite" aria-label="3 direct reports">1</span>
```

Comment:
We shouldn't use direct reports here since that is static
```suggestion
<span class="wdc-count-badge" aria-live="polite" aria-label="3 new notifications">1</span>
```